content/v2/openapi.yml

```diff
@@ -4112,7 +4100,7 @@ components:
             properties:
               period:
                 type: integer
-              preimum_price:
+              premium_price:
```

Comment:
Isn't premium_price supposed to be removed?

Reply:
Nope, this is the parameter. Not the response field.

Reply:
Ah, ok. 👍